src/daffodilDebugger/debugger.ts

```diff
@@ -145,16 +145,32 @@ export async function getDebugger(
 
     // Get schema file before debugger starts to avoid timeout
     if (config.schema.path.includes('${command:AskForSchemaName}')) {
-      config.schema.path = await vscode.commands.executeCommand(
+      const schemaPath = await vscode.commands.executeCommand(
         'extension.dfdl-debug.getSchemaName'
       )
+
+      // Stop execution if schemaPath is null, undefined, or empty string
+      if (!schemaPath) {
+        vscode.window.showErrorMessage('Schema path not provided.')
```

Comment:
I think this should be removed - right now, if I press Esc on an input, I get two error messages, but I should only ever see one.